.github/workflows/enos-test-ui.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: Vault UI Tests
+
+on:
+  workflow_call:
+    inputs:
+      test_filter:
+        type: string
+        description: "A filter to limit the ui tests to. Will be appended to the ember test command as '-f=<filter>'"
+        required: false
+      storage_backend:
+        type: string
+        description: "The storage backend to use, either 'raft' or 'consul'"
+        default: raft
+
+jobs:
+  run-ui-tests:
+    name: Run UI Tests
+    runs-on: custom-linux-xl-vault-latest
+    timeout-minutes: 60
+    env:
+      GITHUB_TOKEN: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
+      # Pass in enos variables
+      ENOS_VAR_aws_region: us-east-1
+      ENOS_VAR_aws_ssh_keypair_name: ${{ github.event.repository.name }}-ci-ssh-key
+      ENOS_VAR_aws_ssh_private_key_path: ./support/private_key.pem
+      ENOS_VAR_tfc_api_token: ${{ secrets.TF_API_TOKEN }}
+      ENOS_VAR_terraform_plugin_cache_dir: ./support/terraform-plugin-cache
+      ENOS_VAR_vault_license_path: ./support/vault.hclic
+      NODE_VERSION: 16
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
nit: there are newer major versions of `actions/checkout` and some other actions you've included here, should we update those?

Reply:
good idea, updated all the actions to the latest versions for those that had newer versions.